src/ProjectOrigin.Registry/TransactionProcessor/TransactionProcessorWorker.cs

```diff
@@ -46,17 +46,17 @@ await _channel.Channel.QueueDeclareAsync(
                 durable: true,
                 exclusive: false,
                 autoDelete: false,
-                arguments: null
-                );
+                arguments: null,
+                cancellationToken: cancellationToken);
 
             await _channel.Channel.BasicQosAsync(
                     prefetchSize: 0,
                     prefetchCount: 5,
-                    global: false
-                    );
+                    global: false,
+                    cancellationToken: cancellationToken);
 
             var consumer = new AsyncEventingBasicConsumer(_channel.Channel);
-            consumer.Received += Consumer_Received;
+            consumer.ReceivedAsync += Consumer_Received;
```

Comment:
Do we need to await something?

Reply:
Was thinking if we need to put the keyword await someplace when the Received is Async now?